src/quest_manager/templates/quest_manager/tab_quests_submission.html

```diff
@@ -66,6 +66,18 @@ <h4 class="panel-title">
               </p>
           </li>
         </ul>
+        <!-- will load a student quick reply form for submissions that have been returned or handed in this course -->
+        {% if s.is_returned or completed and not past %}
```

Comment:
as above, including this for past quests is fine.

Reply:
"not past" clause removed.